Patches.cs

```diff
@@ -88,5 +88,96 @@ public static class Patches
+			new Patch("__ZN14AchievementGui27updateInGameLongEnoughLabelEv", Offset: 0x25,
+				Target: [
+					0x48, 0x8B, 0x48, 0x20,                   // mov     rcx, [rax + 0x20] 
+					0x48, 0x8B, 0xB1, 0x70, 0x01, 0x00, 0x00, // mov     rsi, [rcx + 0x170]
+					0x48, 0x89, 0xF1,                         // mov     rcx, rsi
+					0x48, 0xD1, 0xE9,                         // shr     rcx, 0x01
+					0x48, 0x8D, 0x9F, 0xA8, 0x15, 0x00, 0x00, // lea     rbx, [this + 0x15a8]
+					0x48, 0x3B, 0x88, 0x78, 0x02, 0x00, 0x00, // cmp     rcx, [rax + 0x278]
+				],
+				Replacement: [
+					0x48, 0x8B, 0x48, 0x20,                   // mov     rcx, [rax + 0x20] 
+					0x48, 0x8B, 0xB1, 0x70, 0x01, 0x00, 0x00, // mov     rsi, [rcx + 0x170]
+					0x48, 0x89, 0xF1,                         // mov     rcx, rsi
+					0x48, 0x31, 0xc9,                         // xor     rcx, rcx
+					0x48, 0x8D, 0x9F, 0xA8, 0x15, 0x00, 0x00, // lea     rbx, [this + 0x15a8]
+					0x48, 0x3B, 0x88, 0x78, 0x02, 0x00, 0x00, // cmp     rcx, [rax + 0x278]
+				]
+			),
+		];
+
+		public static Patch[] Mac_aarch64 = [
+			// AchievementStats::allowed
+			new Patch("__ZN16AchievementStats7allowedERK3Map9NamedBoolI21IsLocalAchievementTagEbb", Offset: 0x128,
+				Target: [
+					0x09, 0x3D, 0x41, 0xF9, // ldr     x9,[x8, #0x278]
+					0x08, 0x11, 0x40, 0xF9, // ldr     x8,[x8, #0x20]
+					0x08, 0xB1, 0x40, 0xF9, // ldr     x8,[x8, #0x160]
+					0x3F, 0x05, 0x48, 0xEB, // cmp     x9,x8, LSR #0x1
+				],
+				Replacement: [
+					0x09, 0x3D, 0x41, 0xF9, // ldr     x9,[x8, #0x278]
+					0x08, 0x11, 0x40, 0xF9, // ldr     x8,[x8, #0x20]
+					0xe8, 0x03, 0x1F, 0xAA, // mov     x8,xzr
+					0x3F, 0x05, 0x48, 0xEB, // cmp     x9,x8, LSR #0x1
+				]
+			),
+			// AchievementGui::updateInGameLongEnoughLabel
+			new Patch("__ZN14AchievementGui27updateInGameLongEnoughLabelEv", Offset: 0x18,
+				Target: [
+					0x08, 0x64, 0x43, 0xf9, // ldr     x8,[x0, #0x6c8]
+					0x09, 0x3d, 0x41, 0xf9, // ldr     x9,[x8, #0x278]
+					0x08, 0x11, 0x40, 0xf9, // ldr     x8,[x8, #0x20]
+					0x00, 0xb1, 0x40, 0xf9, // ldr     x0,[x8, #0x160]
+					0x08, 0xb5, 0x82, 0x52, // mov     w8,#0x15a8
+					0xb3, 0x02, 0x08, 0x8b, // add     x19,x21,x8
+					0x3f, 0x05, 0x40, 0xeb, // cmp     x9,x0, LSR #0x1
+				],
+				Replacement: [
+					0x08, 0x64, 0x43, 0xf9, // ldr     x8,[x0, #0x6c8]
+					0x09, 0x3d, 0x41, 0xf9, // ldr     x9,[x8, #0x278]
+					0x08, 0x11, 0x40, 0xf9, // ldr     x8,[x8, #0x20]
+					0x00, 0x00, 0x80, 0xD2, // mov     x0,#0x0
+					0x08, 0xb5, 0x82, 0x52, // mov     w8,#0x15a8
+					0xb3, 0x02, 0x08, 0x8b, // add     x19,x21,x8
+					0x3f, 0x05, 0x40, 0xeb, // cmp     x9,x0, LSR #0x1
+				]
+			),
+		];
+
+		public static Dictionary<Architecture, Patch[]> MacPatchSets = new() {
+			[Architecture.X64] = Mac_x86,
+			[Architecture.Arm64] = Mac_aarch64
+		};
+
+		public static Dictionary<OSPlatform, Dictionary<Architecture, Patch[]>> PlatformPatchSets = new() {
```

Comment:
Do you need to expose each of the `<Platform>PatchSets` dicts above? I think it looks better to create them inline in this nested dictionary here.